CHANGELOG-1.3.md

```diff
@@ -0,0 +1,38 @@
+# CHANGELOG of v1.3
+
+Previous change logs can be found at [CHANGELOG-1.2](https://github.com/opencurve/curve/blob/master/CHANGELOG-1.2.md)
+
+## new features
+
+- [Support discard to garbage collection.](https://github.com/opencurve/curve/pull/189)
+- [QoS of client.](https://github.com/opencurve/curve/pull/268)
+- [QoS of SnapshotCloneServer.](https://github.com/opencurve/curve/pull/303)
+- [Recover file in recycleBin.](https://github.com/opencurve/curve/pull/259)
+- [Clean chunks recycled to chunkfile pool background.](https://github.com/opencurve/curve/pull/322)
```

Comment:
fill chunks  in chunkfile pool with 0 in the background.

Reply:
> fill chunks in chunkfile pool with 0 in the background.

done